types/statsd-client/index.d.ts

```diff
@@ -1,9 +1,11 @@
-// Type definitions for statsd-client v0.1.0
+// Type definitions for statsd-client v0.4.0
 // Project: https://github.com/msiebuhr/node-statsd-client
 // Definitions by: Peter Kooijmans <https://github.com/peterkooijmans>
+//                 Christopher Eck <https://github.com/chrisleck>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
 
-
+import * as express from "express";
```

Comment:
Just a suggestion, but if you're only using `RequestHandler` you can just import that. I think it's more readable that way.

Reply:
I'm using `Request`, `Response`, and `RequestHandler`.